pkg/granted/sso.go

```diff
@@ -16,6 +18,8 @@ import (
 	"gopkg.in/ini.v1"
 )
 
+var defaultProfileNameTemplate = "{{.AccountName | replace \" \" \"-\"}}/{{.RoleName}}"
```

Comment:
To me, normalisation of the .AccountName feels like something that should be done outside the template. So a user is able to write a template like so.

If accountName is already normalised in the available template data then we can drop the requirement for github.com/Masterminds/sprig/v3 unless you can think of more complex use-cases someone may want to use?

```go
type SSOProfile struct {
	// SSO details
	StartUrl  string
	SSORegion string
	// Account and role details
	AccountId   string
	AccountName string
	RoleName    string
}

nameTemplate := "{{ .AccountName }}/{{ .RoleName }}"

or

nameTemplate := "{{ .AccountName }}_{{ .RoleName }}"
```

Reply:
I agree that normalization can happen before template is applied. For sprig I added it mainly for the replace but it gives many functions to manipulate text that gives more freedom to generate profile name. Personally I don't need it. Happy to remove that dependency from this PR.